l10n_it_invoices_data_communication/README.rst

```diff
@@ -47,16 +47,16 @@ Known issues / Roadmap
 
 Qualora questo modulo venisse installato in un DB dove ``l10n_it_comunicazione_dati_iva`` è già installato, è necessario eseguire la sequente procedura:
 
-1. Installare ``openupgradelib``
```

Comment:
Meglio mantenere i numeri visto che sono operazioni in sequenza:
```
#. Installare ``openupgradelib``
#. Lanciare odoo con il paramentro ``shell``
#. Eseguire i seguenti comandi

    >>> from openupgradelib import openupgrade
    >>> openupgrade.update_module_names(env.cr, [('l10n_it_comunicazione_dati_iva', 'l10n_it_invoices_data_communication'),], merge_modules=False,)
    >>> env.cr.commit()

#. Riavviare odoo
#. Aggiornare ``l10n_it_invoices_data_communication``
```

Reply:
Come si fa a far comparire 4 per `Riavviare odoo`?

Reply:
@eLBati se il blocco di codice è indentato correttamente non interrompe la sequenza

Reply:
fatto